docs/modules/ROOT/pages/cops_lint.adoc

```diff
@@ -1229,6 +1229,44 @@ end
 
 * https://rubystyle.guide#no-flip-flops
 
+== Lint/FloatComparison
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| No
+| 0.89
+| -
+|===
+
+This cop checks for the presence of precise comparison of floating point numbers.
+
+Floating point values are inherently inaccurate, and comparing them for exact equality
+is almost never the desired semantics. Comparison via the `==/!=` operators checks
+floating-point value representation to be exactly the same, which is very unlikely
+if you perform any arithmetic operations involving precision loss.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+x == 0.1
+x != 0.1
+
+# good
+(x - 0.1).abs < Float::EPSILON
```

Comment:
There's also BigDecimal that we can recommend. It seems to me that using it makes the code much simpler to understand.